openshift/templates/pims-api/test.yaml

```diff
@@ -0,0 +1,126 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: pims-api-test
```

Comment:
Just a minor comment on naming. We have been using the convention (component)-(env) for naming builds in openshift. This however is for running unit tests, not the Pims API for the test environment. Any ideas on how we cod rename this better?

Reply:
I've renamed